### PR TITLE
Update RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc

### LINKED ISSUE #228: Proof MYSUBS
Proof read transcribed code of [MYSUBS](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/MYSUBS.agc) against scans

Lines: 93
Page: 0999—1000

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### LINKED ISSUE #248: Proof RCS-CSM_DAP_EXECUTIVE_PROGRAMS
Proof read transcribed code of [RCS-CSM_DAP_EXECUTIVE_PROGRAMS](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc) against scans

Lines: 86
Page: 1037—1038

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### PR DESCRIPTION
**Pull request by g3r4n**
RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc : 
Page [1037](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1037.jpg) and [1038](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1038.jpg)

Closes #248 

MYSUBS.agc : 
Page [999](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0999.jpg) , [1000](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1000.jpg) and [1001](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1001.jpg)

Closes #228

### CODE REVIEW

**Comment by g3r4n** [comment]
Sorry for mess. I will make branch for others 😄

File: Comanche055/MYSUBS.agc [hunk 1]
```diff
@@ -74,19 +68,14 @@ DPADD		DAS	KMPAC
...
 		AD	POSMAX		# KMPAC LESS THAN 0
 		TCF	TSK
-
 DPADD2+		AD	NEGMAX		# CAN NOT OVERFLOW
 		TS	KMPAC 	+1
 		CA	NEGMAX		# UPPER WAS = 0
 		TCF	TSK
-
 # Page 1001 (empty page)
-
```
Comment (wopian) [inline Comanche055/MYSUBS.agc]: :exclamation: A new line at the end of the file is required

File: Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc [hunk 2]
```diff
@@ -30,7 +30,6 @@
 
 # Page 1037
 # CALCULATION OF  AMGB, AMBG  ONCE EVERY SECOND
-#
```
Comment (wopian) [inline Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc]: :exclamation: Intended line break

File: Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc [hunk 3]
```diff
@@ -41,10 +40,10 @@
 #
 # WHERE PHI AND PSI ARE CDU ANGLES
 
+
```
Comment (wopian) [inline Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc]: :+1: Approved

File: Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc [hunk 4]
```diff
@@ -41,10 +40,10 @@
...
 		BANK	20
 		SETLOC	DAPS8
 		BANK
-
```
Comment (wopian) [inline Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc]: :exclamation: Intended line break

File: Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc [hunk 5]
```diff
@@ -53,7 +52,6 @@ AMBGUPDT	CA	FLAGWRD6	# CHECK FOR RCS AUTOPILOT
 		MASK	BIT14		# IF NOT RCS, EXIT
 		EXTEND
 		BZF	ENDOFJOB	# TO PROTECT TVC DAP ON SWITCHOVER
-
```
Comment (wopian) [inline Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc]: :exclamation: Intended line break

File: Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc [hunk 6]
```diff
@@ -82,4 +80,3 @@ AMBGUPDT	CA	FLAGWRD6	# CHECK FOR RCS AUTOPILOT
 		TS	AMGB7
 		TCF	ENDOFJOB
 QUADANGL	DEC	660		# = 7.25 DEGREES
-
```
Comment (wopian) [inline Comanche055/RCS-CSM_DAP_EXECUTIVE_PROGRAMS.agc]: :exclamation: A new line at the end of file is required

**Comment by g3r4n** [comment]
I will open proper PR for each issue with the suitable correction.

**Pull request closed without merging**